src/components/m-datepicker/index.js

```diff
@@ -9,8 +13,26 @@ import Datepicker from './js/datepicker';
 
 import getAttribute from '../../js/get-attribute';
 
+const startType = PropTypes.oneOfType([
+  PropTypes.number,
+  PropTypes.oneOf([TODAY]),
+]);
+
 class AXADatepicker extends BaseComponentGlobal {
```

Comment:
duplicated class name tracked in #589

Reply:
imho we should prefix all classes with M and A apart of organisms